services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -631,11 +642,8 @@ private boolean createOrVerifyInternalStore(
       }
 
       if (!store.isHybrid()) {
-        UpdateStoreQueryParams updateStoreQueryParams;
-        updateStoreQueryParams = new UpdateStoreQueryParams();
         updateStoreQueryParams.setHybridOffsetLagThreshold(100L);
         updateStoreQueryParams.setHybridRewindSeconds(TimeUnit.DAYS.toSeconds(7));
```

Comment:
These should be applied only if the `updateStoreQueryParams` from the argument doesn't specify these configs. Or, the `updateStoreQueryParams` argument can be changed to `activeActiveReplicationEnabled`, and then we decide the DRP based on this argument

Reply:
That's okay. So, the changes about hybrid configs being applied only if the `updateStoreQueryParams` from the argument doesn't specify these configs need to be made

Reply:
For here, the check is: if the store is not hybrid, which means these arguments are not specified. Is that what you mean?

Reply:
I meant to say that if the UpdateStoreQueryParams from the argument specifies that `HybridOffsetLagThreshold` should be 500, then the implementation should respect that instead of overwriting it with 100

Reply:
I see, we can add if check here to see if these fields existed or not. If it existed, we do not overwrite it.